Marlin/src/lcd/ultralcd.cpp

```diff
@@ -3678,7 +3682,10 @@ void kill_screen(const char* lcd_msg) {
   void lcd_control_filament_menu() {
     START_MENU();
     MENU_BACK(MSG_CONTROL);
-
+    #if ENABLED(ADVANCED_PAUSE_SPOOL_SWAP)//steeve
+	  MENU_ITEM_EDIT(bool,MSG_SWAP_SPOOL_ON, &swap_spool_enabled);
+	  MENU_ITEM_EDIT(int3,MSG_SWAP_SPOOL_STOP, &swap_spool_stop,active_extruder,FIL_RUNOUT_SENSORS);
+    #endif																				 
```

Comment:
Just two menus
ONE to activate spool swap
One to set the max number of swap allowed
'i didn't make a limiter code , to limit this data if no sensors available or no extruders'
The value is applyied , but , when go in pause , if no enough hardware , it will be turned OFF